oneflow/core/functional/impl/math_functor.cpp

```diff
@@ -37,6 +38,35 @@ limitations under the License.
 namespace oneflow {
 namespace one {
 namespace functional {
+namespace {
+Maybe<std::vector<int32_t>> check_axis(int32_t naxis, const std::vector<int32_t>& axis) {
```

Comment:
这里不需要返回maybe，直接返回vector吧
```suggestion
Maybe<std::vector<int32_t>> check_axis(const int32_t &ndim, const std::vector<int32_t>& axis) {
```

Reply:
收到